GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -1359,7 +1359,8 @@ void DoCommit()
                         noVerifyToolStripMenuItem.Checked,
                         gpgSignCommitToolStripComboBox.SelectedIndex > 0,
                         toolStripGpgKeyTextBox.Text,
-                        Staged.IsEmpty);
+                        Staged.IsEmpty,
+                        amend);
```

Comment:
I do not want to change this by default. 
GE should follow the Git defaults when it makes sense, to work predictably.
If the change is minor, then the original author should be preserved.

So unfortunately this need a GUI control, probably a checkbox appearing when amend is checked.

Reply:
I have added an additional checkbox. The enabled state of this checkbox is toggled with the checked state of amend.

![grafik](https://user-images.githubusercontent.com/7032094/147751788-40329e08-114a-44ac-914b-9753895fde45.png)